src/poetry/console/commands/cache/list.py

```diff
@@ -11,10 +11,8 @@ class CacheListCommand(Command):
     description = "List Poetry's caches."
 
     def handle(self) -> int | None:
-        from poetry.locations import REPOSITORY_CACHE_DIR
-
-        if os.path.exists(str(REPOSITORY_CACHE_DIR)):
-            caches = sorted(REPOSITORY_CACHE_DIR.iterdir())
+        if os.path.exists(str(self.poetry.config.repository_cache_directory)):
```

Comment:
`repository_cache_directory` is `Path`, so we can do: 
```suggestion
        if self.poetry.config.repository_cache_directory.exists():
```
Looks much cleaner imho

Reply:
Good catch.